### PR TITLE
V0.5.0 test

### PR DESCRIPTION
**Pull request by chiahaoliu**
modified code after testing at XPD

### CODE REVIEW

File: xpdacq/glbl.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
     BASE_DIR = os.getcwd()
 else:
     #BASE_DIR = os.path.expanduser('~/')
-    BASE_DIR = os.path.abspath('/direct/XF28ID1/pe1_data')
+    BASE_DIR = os.path.abspath('/direct/XF28ID1/pe1_data/UserArea/XPDhome/xpdUser')
```
Comment (sbillinge) [inline xpdacq/glbl.py]: this should not have xpdUser on it

File: xpdacq/glbl.py [hunk 2]
```diff
@@ -29,7 +29,8 @@
 if simulation:
     BASE_DIR = os.getcwd()
 else:
-    BASE_DIR = os.path.expanduser('~/')
+    #BASE_DIR = os.path.expanduser('~/')
+    BASE_DIR = os.path.abspath('/direct/XF28ID1/pe1_data/UserArea/XPDhome/xpdUser')
```
Comment (sbillinge) [inline xpdacq/glbl.py]: no xpdUser here

File: xpdacq/utils.py [hunk 3]
```diff
@@ -277,7 +277,7 @@ def load(self, saf_num):
 
         self.pd_dict = pd.read_excel(os.path.join(self.src_dir,
                                                   xl_f.pop()),
-                                     skiprows=0)
+                                     skiprows=[1])
```
Comment (sbillinge) [inline xpdacq/utils.py]: I will assume it is correct, but please check.  Does this skip row 1 or all rows up to 1?  I would rather expect to see `skiprows=[:1]`

File: xpdacq/glbl.py (hunk 2 above)
Comment (chiahaoliu) [inline xpdacq/glbl.py]: updated with commit just pushed

File: xpdacq/utils.py (hunk 3 above)
Comment (chiahaoliu) [inline xpdacq/utils.py]: we need first row to be a column name so I only skip second row

**Comment by CJ-Wright** [comment]
You need to propagate the refactor of `import_sample` -> `import_sample_info` into the tests.

**Comment by chiahaoliu** [comment]
will separate in small PR

**Comment by CJ-Wright** [comment]
Awesome, thanks

**Comment by CJ-Wright** [comment]
@chiahaoliu Do you mind rebasing onto master if this PR is still going to be used.

**Comment by chiahaoliu** [comment]
@CJ-Wright this PR will not be used, instead, it is served as bookkeeping purpose. As described above, I will break it into several small ones. Now I am 2 small PR away from completing this PR, if you don't mind, can I leave it here? Of course, I am happy to rebase as well.

**Comment by CJ-Wright** [comment]
Can we close this then?

**Comment by chiahaoliu** [comment]
close as it will be separated into small PRs

**Pull request closed without merging**